x/interchainstaking/keeper/delegation.go

```diff
@@ -280,28 +280,34 @@ func (k *Keeper) MakePerformanceDelegation(ctx sdk.Context, zone *types.Zone, va
 }
 
 func (k *Keeper) FlushOutstandingDelegations(ctx sdk.Context, zone *types.Zone, delAddrBalance sdk.Coin) error {
-	var amount sdk.Coins
+	var pendingAmount sdk.Coins
 	exclusionTime := ctx.BlockTime().AddDate(0, 0, -1)
-
+	var coinsToFlush sdk.Coins
 	k.IterateZoneReceipts(ctx, zone, func(_ int64, receiptInfo types.Receipt) (stop bool) {
-		if receiptInfo.FirstSeen.After(exclusionTime) && receiptInfo.Completed == nil {
-			amount = amount.Add(receiptInfo.Amount...)
+		if (receiptInfo.FirstSeen.After(exclusionTime) || receiptInfo.FirstSeen.Equal(exclusionTime)) && receiptInfo.Completed == nil {
+			pendingAmount = pendingAmount.Add(receiptInfo.Amount...)
 		}
 		return false
 	})
 
-	coins := sdk.NewCoins(delAddrBalance).Sub(amount...)
+	var hasNeg bool
 
-	if coins.IsAnyNegative() || coins.IsZero() {
+	coinsToFlush, hasNeg = sdk.NewCoins(delAddrBalance).SafeSub(pendingAmount...)
```

Comment:
We can just make this 

```go
	coinsToFlush, hasNeg := sdk.NewCoins(delAddrBalance).SafeSub(pendingAmount...)
```

and remove the `var` declarations

Reply:
resolved in #445